docs/getting-started/linux.rst

```diff
@@ -81,4 +78,4 @@ The following steps will install `dnvm <https://github.com/aspnet/home#running-a
 If you have trouble installing dnvm, consult this `Getting Started guide <http://dotnet.github.io/core/getting-started/>`_.
 
 
-.. include:: xplat.rst
\ No newline at end of file
+.. include:: x-plat/guide.rst
```

Comment:
assuming this fixes the build warning about the file not being included in a TOC?